pep-0008.txt

```diff
@@ -705,7 +705,7 @@ letter (never alter the case of identifiers!).
 Block comments generally consist of one or more paragraphs built out of
 complete sentences, with each sentence ending in a period.
 
-You should use two spaces after a sentence-ending period in multi-
+You should use one or two spaces after a sentence-ending period in multi-
 sentence comments, except after the final sentence.
```

Comment:
```suggestion
You should use one or two spaces after a sentence-ending period in
multi-sentence comments, except after the final sentence.
```
Noticed this doesn't get rendered correctly:
![Screen Shot 2023-02-24 at 4 27 37 PM](https://user-images.githubusercontent.com/906600/221325006-aa119356-55ec-4297-99dd-e2cba3adc1fd.png)